oai_agents/common/learner.py

```diff
@@ -0,0 +1,29 @@
+class Learner:
```

Comment:
LearnerType should be a class inside `class Learner` for better clarity.
Something like:

```
Class Learner:
    Class LearnerType:
              Helper = 'helper' 
              ....

    def __init__(self, learner_type):
          self.learner_type = learner_type
```

Reply:
I prefer to have them outside Learner, but still in the same function. Therefore, when we call them, we just call learnertype like we call teamtypes. 
I hope this works for you. If you prefer other ways, I would like to discuss with you!!

Reply:
This looks good as well!